spec/classes/redis_sentinel_spec.rb

```diff
@@ -136,6 +136,7 @@
 dir /tmp/redis
 daemonize #{facts[:osfamily] == 'RedHat' ? 'no' : 'yes'}
 pidfile #{pidfile}
+protected-mode yes
```

Comment:
You need to honor `$redis::sentinel::protected_mode`. The version on Ubuntu 16.04 is too old.

Reply:
mhm, we default to true in init.pp. you suggest it should be false for Ubuntu 16.04?

Reply:
not sure if this fix is what you had in mind, but it works now.